alf/algorithms/config.py

```diff
@@ -225,7 +226,8 @@ def __init__(self,
                 help prevent a dead loop in some deterministic environment like
                 Breakout. Only used for evaluation.
             eval_uncertainty (bool): whether to evaluate uncertainty after training.
-            num_eval_episodes (int) : number of episodes for one evaluation.
+            num_eval_episodes (int): number of episodes for one evaluation.
+            num_eval_steps (int): number of environmental steps for evaluation.
```

Comment:
It may make more sense to let the user specify the number of steps during invoking the on-demand evaluation function.

Reply:
This PR (#1743) was implemented earlier and is mainly served for the purpose of enabling the  evaluation (the auto evaluation parallel to the training) even without the concept of ``episode`` on the env side.


The online eval request feature was something comes later (#1744).